src/main/java/net/greatstart/controllers/ProjectController.java

```diff
@@ -72,7 +76,7 @@ public ModelAndView showMyProjects(Principal principal) {
         return model;
     }
 
-    @GetMapping(value = "/{id}")
+    /*@GetMapping(value = "/{id}")
     public ModelAndView showProject(@PathVariable Long id) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)